packages/kernel-5.10/kernel-5.10.spec

```diff
@@ -42,6 +48,11 @@ Requires: %{_cross_os}microcode-licenses
 Requires: %{name}-modules = %{version}-%{release}
 Requires: %{name}-devel = %{version}-%{release}
 
+# Pull in platform-dependent modules.
+%if "%{_cross_arch}" == "x86_64"
+Requires: (%{name}-modules-neuron if (%{_cross_os}variant-platform(aws) without %{_cross_os}variant-flavor(nvidia)))
```

Comment:
Should this be more restrictive, or is it OK to get the Neuron kernel module in FIPS variants?

Reply:
> Should this be more restrictive, or is it OK to get the Neuron kernel module in FIPS variants?

Unlike NVIDIA, Neuron doesn't provide a PKCS#11 interface to userspace, so it seems fine to include it on FIPS variants.